tests/filesTest.js

```diff
@@ -1398,115 +1541,58 @@ describe('Main: Currently testing files management,', function () {
-        const oc = createOwncloud()
+        const oc = createOwncloud(config.testUser, config.testUserPassword)
         await oc.login()
-        return oc.systemTags.createTag({ name: newTagName })
+        return oc.systemTags.createTag({ name: tagToCreate })
           .catch(error => {
             expect(error).toBe(null)
           })
       })
     })
 
-    it('checking method: file info', async function () {
-      const newFile = testFolder + '/' + testFile
-      const provider = createProvider(true, true)
-      await getCapabilitiesInteraction(provider)
-      await getCurrentUserInformationInteraction(provider)
-
-      const getFileInfoBy = data => {
-        return {
-          status: 207,
-          headers: applicationXmlResponseHeaders,
-          body: new XmlBuilder('1.0', '', 'd:multistatus').build(dMultistatus => {
-            dMultistatus.setAttributes({
-              'xmlns:d': 'DAV:',
-              'xmlns:s': 'http://sabredav.org/ns',
-              'xmlns:oc': 'http://owncloud.org/ns'
-            })
-            dMultistatus.appendElement('d:response', '', dResponse => {
-              dResponse.appendElement(
-                'd:href', '',
-                `/remote.php/${data === fileId ? 'webdav' : 'dav/files/' + config.adminUsername}/${testFolder}/${testFile}`
-              )
-                .appendElement('d:propstat', '', dPropstat => {
-                  dPropstat.appendElement('d:prop', '', dProp => {
-                    dProp
-                      .appendElement('oc:fileid', '', fileId)
-                  })
-                    .appendElement('d:status', '', 'HTTP/1.1 200 OK')
-                })
-            })
-          })
-        }
-      }
-
-      await provider
-        .uponReceiving(`as '${username}', a PROPFIND request to file info, fileId`)
-        .withRequest({
-          method: 'PROPFIND',
-          path: webdavPath(`${testFolder}/${testFile}`),
-          headers: {
-            ...validAuthHeaders,
-            ...applicationXmlResponseHeaders
-          },
-          body: new XmlBuilder('1.0', '', 'd:propfind').build(dPropfind => {
-            dPropfind.setAttributes({ 'xmlns:d': 'DAV:', 'xmlns:oc': 'http://owncloud.org/ns' })
-            dPropfind.appendElement('d:prop', '', dProp => {
-              dProp.appendElement('oc:fileid', '', '')
-            })
-          })
-        })
-        .willRespondWith(getFileInfoBy('fileId'))
-
-      return provider.executeTest(async () => {
-        const oc = createOwncloud()
-        await oc.login()
-        return oc.files.fileInfo(newFile, ['{http://owncloud.org/ns}fileid'])
-          .then(fileInfo => {
-            expect(fileInfo.getProperty('{http://owncloud.org/ns}fileid')).toEqual('123456789')
-          }).catch(error => {
-            expect(error).toBe(null)
-          })
-      })
-    })
-
     it('checking method: tag file', async function () {
       const provider = createProvider(true, true)
```

Comment:
should pass now in oc10, can you set it to `const provider = createProvider(false, true)`